README.md

```diff
@@ -208,14 +208,30 @@ Mac: Install autotools using [MacPorts](https://www.macports.org/install.php)
 port install automake autoconf libtool pkgconfig
 ```
 
-Windows: Install autotools using [MSYS2](http://www.msys2.org/)
+Windows: The [MSYS2](http://www.msys2.org/) does not support Node.js anymore. We have to find another way to build it using the Windows platform.
+
+The old (obsolete) instructions:
+
+Install autotools using [MSYS2](http://www.msys2.org/)
 ```bash
 pacman -S make autoconf automake libtool pkgconfig
 ```
 
-[Install Node.js](https://nodejs.org/en/download/)
+[Install Node.js 16 LTS](https://nodejs.org/dist/v16.13.2/node-v16.13.2.pkg)
+
+Use [nvm](https://github.com/nvm-sh/nvm#install--update-script) in order to quickly install and switch between the different Node.js versions.
+
+Note: Node 17 does not support the required openssl functionality.
```

Comment:
Can we bump to node 18 which is LTS?

Reply:
Unfortunately, we can't.
There's an [ERR_OSSL_EVP_UNSUPPORTED error](https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported).

When setting the `openssl-legacy-provider` option - I've got an error from emscripten:
```
/Users/iserba/dev/emsdk/node/12.9.1_64bit/bin/node: --openssl-legacy-provider is not allowed in NODE_OPTIONS
```
So downgrading to the node v16 is the only way I found

Reply:
Unfortunately, we can't.
There's an [ERR_OSSL_EVP_UNSUPPORTED error](https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported).

When setting the `openssl-legacy-provider` option - I've got an error from emscripten:
```
/Users/iserba/dev/emsdk/node/12.9.1_64bit/bin/node: --openssl-legacy-provider is not allowed in NODE_OPTIONS
```
So downgrading to the node v16 is the only way I found